test/server/cards/02_SHD/units/WildRancor.spec.ts

```diff
@@ -0,0 +1,29 @@
+describe('Wild Rancor', function() {
+    integration(function(contextRef) {
+        describe('Wild Rancor\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['wild-rancor'],
+                        groundArena: ['battlefield-marine'],
```

Comment:
Let's add a leader for this check just to be sure that the method for getting all units in an arena is correctly getting leaders, I don't know if we've tested that yet